source/main/physics/Beam.cpp

```diff
@@ -927,11 +920,8 @@ Vector3 Actor::calculateCollisionOffset(Vector3 direction)
                         nb->AbsPosition - collision_offset,
                         actor->ar_collision_range * 3.0f);
 
-                    if (!m_intra_point_col_detector->hit_list.empty())
-                    {
-                        collision = true;
+                    if (collision = !m_intra_point_col_detector->hit_list.empty())

```

Comment:
I don't like this sort of C-gymnastics. Does it affect performance at all (since it's in a 'performance' commit)?

Reply:
It doesn't affect the performance.